### PR TITLE
imx6ull: implement nor storage over qspi

### PR DESCRIPTION
**Pull request by badochov**
JIRA: RTOS-320

## Description
Almost a copy of nor flash driver from plo.
imx6ull: implement nor storage over qspi

## Motivation and Context
This driver is neeted to allow jffs2 rootfs on this target.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: spi/imx6ull-qspi/qspi.c [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+		return -ENOMEM;
+	}
+
+	qspi->slFlashSz[0] = qspi->base[QSPI_SFA1AD] - qspi->ahbAddr;
+	qspi->slFlashSz[1] = qspi->base[QSPI_SFA2AD] - qspi->base[QSPI_SFA1AD];
+	qspi->slFlashSz[2] = qspi->base[QSPI_SFB1AD] - qspi->base[QSPI_SFA2AD];
+	qspi->slFlashSz[3] = qspi->base[QSPI_SFB2AD] - qspi->base[QSPI_SFB1AD];
+
+	return EOK;
+}
```
Comment (agkaminski) [inline spi/imx6ull-qspi/qspi.c:301]:
```suggestion
}

```

**Review by agkaminski — COMMENTED** [review]
Minor nitpicks, otherwise LGTM. It would be good to confirm no regression on DTR project (I don't see a potential for a problem, but better safe than sorry)

**Review by agkaminski — APPROVED** [review]
LGTM

**Comment by badochov** [comment]
No regression on DTR, merging.

**Pull request merged**